lib/config/defaults.js

```diff
@@ -1435,6 +1435,14 @@ const getResolveDefaults = ({ cache, context, targetProperties, mode }) => {
 		extensions: [...jsExtensions]
 	});
 
+	/** @type {function(): ResolveOptions} */
+	const styleDeps = () => ({
+		conditionNames: ["style"],
+		mainFields: ["style", "main", "..."],
+		mainFiles: ["index", "..."],
```

Comment:
I don't think we need it here, becaue it will resolve `index.css` when you have `@import "./directory"`, it was in css-loader only for legacy purposes, we don't need to put it here

Reply:
So we need a test case here too